Sources/GRPCCodeGen/Internal/Translator/TypealiasTranslator.swift

```diff
@@ -111,6 +128,32 @@ extension TypealiasTranslator {
     let methodDescriptorsDeclaration = self.makeMethodDescriptors(for: service)
     methodsEnum.members.append(methodDescriptorsDeclaration)
 
+    // Create the static service descriptor property.
+    let serviceDescriptorInitialization = Expression.functionCall(
+      FunctionCallDescription(
+        calledExpression: .identifierType(.member("ServiceDescriptor")),
+        arguments: [
+          FunctionArgumentDescription(
+            label: "package",
+            expression: .literal(service.namespace.base)
+          ),
+          FunctionArgumentDescription(
+            label: "service",
+            expression: .literal(service.name.base)
+          ),
+        ]
+      )
+    )
+
+    let staticServiceDescriptorProperty = VariableDescription(
+      accessModifier: self.accessModifier,
+      isStatic: true,
+      kind: .let,
+      left: .identifierPattern("serviceDescriptor"),
+      right: serviceDescriptorInitialization
+    )
```

Comment:
Can we extract this into a helper? Otherwise these code-gen methods get unwieldy.

Reply:
True!